.github/workflows/helm-ci.yml

```diff
@@ -7,9 +7,43 @@ jobs:
     uses: grafana/helm-charts/.github/workflows/linter.yml@main
     with:
       filter_regex_include: .*operations/helm/.*
+
   call-lint-test:
     uses: grafana/helm-charts/.github/workflows/lint-test.yaml@main
     with:
       ct_configfile: operations/helm/ct.yaml
       ct_check_version_increment: false
       helm_version: v3.8.2
+
+  conftest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.2
+
+      - name: Set up conftest
+        uses: artis3n/setup-conftest@v0
+        with:
+          conftest_version: 0.33.0
+          conftest_wrapper: false # don't wrap conftest so that non-zero exit code fail the job
+
+      - name: Verify
```

Comment:
I think "Verify" and "Run tests" should invoke `make`, so people can check/correct what the static tests find themselves more easily. Conftest has a docker image (https://www.conftest.dev/install/), so it would make sense to try to use that instead of the mimir build image (unless BUILD_IN_CONTAINER is false).

Also if the self tests are quick to run it could be worth combining verify and tests in one target.

Reply:
I've added two Makefile targets. Also updated the build image, so that this can be ran without any external dependencies like helm or conftest. PTAL